readthedocs/search/api.py

```diff
@@ -45,8 +45,10 @@ def get_link(self, obj):
         # Generate an appropriate link for the doctypes that use htmldir,
         # and always end it with / so it goes directly to proxito.
         if doctype in {SPHINX_HTMLDIR, MKDOCS}:
-            new_path = re.sub('(^|/)index.html$', '', path)
-            path = path.rstrip('/') + '/'
+            new_path = re.sub('(^|/)index.html$', '/', path)
+            # docs_url already ends with /,
+            # make sure to not include it twice.
+            path = new_path.lstrip('/')
```

Comment:
This is `lstrip` but the commend says `ends with a /`. We should fix whichever is wrong :)

Reply:
`docs_url` ends with `/`, so path doesn't need to start with `/`

Reply:
Updated the comment to make it more clear